src/map.cpp

```diff
@@ -1785,6 +1785,7 @@ bool ServerMap::saveBlock(MapBlock *block, MapDatabase *db, int compression_leve
 
 void ServerMap::loadBlock(std::string *blob, v3s16 p3d, MapSector *sector, bool save_after_load)
 {
+	ScopeProfiler sp(g_profiler, "ServerMap: deSer block", SPT_AVG);
```

Comment:
I suggest putting this in a tight scope around `block->deSerialize(is, version, true);`

Otherwise it might count `saveBlock` and the reflow scanner too

Reply:
There's no ideal place to put this. Ideally this would also measure the creation of the `istringstream` and the first read from it.